src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/.template.config/ide.host.json

```diff
@@ -37,24 +37,11 @@
   "symbolInfo": [
     {
       "id": "DisableOpenAPI",
-      "name": {
-        "text": "Enable _OpenAPI support"
-      },
-      "description": {
-        "text": "Enables OpenAPI (Swagger) support"
-      },
-      "invertBoolean": true,
-      "isVisible": true,
-      "defaultValue": true
+      "isVisible": true
     },
     {
       "id": "UseMinimalAPIs",
-      "name": {
-        "text": "Use controllers (uncheck to use minimal APIs)"
-      },
-      "invertBoolean": true,
-      "isVisible": true,
-      "defaultValue": true
+      "isVisible": true
```

Comment:
@mkArtakMSFT I've reverted 05758e28dc. That'll be tracked as part of https://github.com/dotnet/aspnetcore/issues/37202.

@phenning any concerns with this approach (no changes to the `WebApi-CSharp` template).

Reply:
So for these, and I'll take a look in the other PR - we should be able to just have the displayName text be the same as we have it currently here. Since displayName is only ussed in Visual Studio and not on the command line (might want to confirm after the change with dotnet new webapi -h).  We'd then remove the name property here in the ide.host.json, but leave the invertBoolean and defaultValue properties.

Reply:
Discussed offline. https://github.com/dotnet/aspnetcore/pull/40105/commits/30a5259960e24dbd3787097b9427ff9f008692a6 should be sufficient for now to avoid a more extensive changeset.